code/game/objects/structures/crates_lockers/largecrate.dm

```diff
@@ -292,6 +292,353 @@
 		while(++m <= num_mags)
 			new new_mag(src)
 
+
+/obj/structure/largecrate/crap
```

Comment:
Use the additions made from https://github.com/cmss13-devs/cmss13/pull/665/ instead of adding new crates.